src/models/step_arguments/data_table.js

```diff
@@ -1,5 +1,31 @@
 import _ from 'lodash'
 
+const truths = ['true', 'TRUE', 'True', 'yes', 'YES', 'Yes', 'y', 'Y', '1']
+const trueFalse = (v) => !!~truths.indexOf(v)
```

Comment:
Okay. I'd prefer the following then:

* Update the implementation to convert the value to lowercase so you `truths` array can at become `['true', 'yes', 'y', '1']`. 
* Add a `falses` array or something likewise. 
* Throw an error if the value is not on of the accepted true / false values